modules/Microsoft.VirtualMachineImages/imageTemplates/.test/advanced/deploy.test.bicep

```
@@ -0,0 +1,105 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.virtualmachineimages.imagetemplates-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'vmitadv'
```

```suggestion
param serviceShort string = 'vmiitadv'
```
Similar too below. I guess it should be this - unless it is too long

---

I updated.